test/spec/helper/InjectSpec.js

```diff
@@ -0,0 +1,110 @@
+import {
+  bootstrapModeler,
+  inject
+} from 'test/TestHelper';
+
+import coreModule from 'lib/core';
+import { expect } from 'chai';
+
+
+describe('helper - inject', function() {
+
+  var diagramXML = require('../../fixtures/bpmn/simple.bpmn');
+
+  beforeEach(bootstrapModeler(diagramXML, {
+    modules: [
+      coreModule
+    ]
+  }));
+
+
+  it('should work with Promise as return value', function() {
+
+    // given
+    var expected = 'resolved';
+
+    // when
+    var test = inject(function(eventBus) {
+
+      expect(eventBus).to.exist;
+
+      return Promise.resolve(expected);
+    });
+
+    // then
+    return test().then(function(result) {
+
+      expect(result).to.eql(expected);
+    });
+  });
+
+
+  it('should work with async function', async function() {
```

Comment:
We do not need to have this test. Returning promises, tested above is equivalent to `async fn()`.